src/pages/terraform/plugin/[[...page]].tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactElement } from 'react'
+import terraformData from 'data/terraform.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+
+const basePath = 'plugin'
+const baseName = 'Plugin'
+const product = terraformData as Product
+const productSlugForLoader = 'terraform-website'
+
+const additionalComponents = {}
```

Comment:
Same question as previous comments about this, just commenting so it's identified in case you end up making changes! 💖